### PR TITLE
Initial Feature Set Proposal

### PR DESCRIPTION
**Pull request by as-cii**
I've created this pull–request to discuss the initial feature–set of captr.io.

Let's see what we come up with! :smile: 

/cc: @exalted

### CODE REVIEW

**Comment by as-cii** [comment]
These are my notes of our last meeting, they could be helpful while writing the features.

> ### 01/02/2014 notes
> - Write in (a subset of) markdown (define better)
> - Create sections
> - Each section has a corresponding video part
> - Can associate only one media element to a section
> - Copy/Paste
> - Timeline is auto–generated
> - Timeline can be reviewed later
> - Video Preview

File: features/captr_editing.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: Captr editing
+  In order to make a captr
+  As a user
+  I want to have a video editor
+
+  Scenario: User creates a basic captr
```
Comment (as-cii) [inline features/captr_editing.feature]: @exalted: do you agree with the name **captr**?

Comment (as-cii) [inline features/captr_editing.feature]: @exalted: I'm not really happy with `basic`. Maybe we can remove it?

**Comment by exalted** [comment]
@as-cii I gotta clear my head before getting into this seriously, in the meantime please don't stop typing… :wink:

File: features/organization.feature [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    Given I am an authenticated user
+    And I am part of an organization
+    When I remove members from it
+    Then they are not part of the organization anymore
+
+  Scenario: User deletes an organization
+    Given I am an authenticated user
+    And I am part of an organization
+    When I delete it
+    Then it gets removed from members' organizations
```
Comment (as-cii) [inline features/organization.feature:28]: @exalted: I am not really sure these last two scenarios are really valuable. What do you think about them?

**Pull request closed without merging**